src/commands/reminder/CreateReminder.ts

```diff
@@ -0,0 +1,170 @@
+                                    .setCustomId("day")
+                                    .setRequired(true),
+                                new TextInputBuilder()
+                                    .setStyle(TextInputStyle.Short)
+                                    .setMaxLength(4)
+                                    .setLabel("Year (xxxx)")
+                                    .setCustomId("year")
+                                    .setRequired(true),
+                                new TextInputBuilder()
+                                    .setStyle(TextInputStyle.Short)
+                                    .setMaxLength(2)
+                                    .setLabel("Hour (1-24)")
+                                    .setCustomId("hour")
+                                    .setRequired(true),
+                                new TextInputBuilder()
+                                    .setStyle(TextInputStyle.Short)
+                                    .setMaxLength(2)
+                                    .setLabel("Minute (1-60)")
+                                    .setCustomId("minute")
+                                    .setRequired(true),
+                            ],
+                            duration: 60 * 1000
+                        }, async (result) => {
+                            await result.deferUpdate();
+                            const month = Number.parseInt(result.fields.getTextInputValue("month"));
+                            const day = Number.parseInt(result.fields.getTextInputValue("day"));
+                            const year = Number.parseInt(result.fields.getTextInputValue("year"));
+                            const hour = Number.parseInt(result.fields.getTextInputValue("hour"));
+                            const minute = Number.parseInt(result.fields.getTextInputValue("minute"));
+
+                            const date = new Date();
+                            date.setMonth(month - 1);
+                            date.setDate(day);
+                            date.setFullYear(year);
+                            date.setHours(hour);
+                            date.setMinutes(minute);
+
+                            let message;
+                            let color: ColorResolvable = "Green";
+                            if (!Number.isNaN(date.getTime())) {
+                                if (date < new Date()){
+                                    message = "Sorry, we can't remind you in the past!";
+                                    color = "Red";
+                                }
+                                else {
+                                    // save the info
+                                    PostGresReminder.insert(ctx.user.id, reminderMsg, date, ctx.channel.id);
+
+                                    message = "Date set! " + `${TimeUtilities.getDiscordTime({ time: date.getTime() })}, `+ "you'll be reminded about: " + `${reminderMsg}`;
```

Comment:
If `reminderMsg` isn't going to be concatenated with anything else, then there's no point in creating a template string with just that variable being interpolated. So, you could do something like

```suggestion
                                    message = "Date set! " + `${TimeUtilities.getDiscordTime({ time: date.getTime() })}, `+ "you'll be reminded about: " + reminderMsg;
```

However, I usually prefer making it explicitly clear what the user's input is. So, you could do something like
```suggestion
                                    message = "Date set! " + `${TimeUtilities.getDiscordTime({ time: date.getTime() })}, `+ "you'll be reminded about: " + StringUtil.codifyString(reminderMsg);
```
which will wrap the reminder text around code blocks.